### PR TITLE
Update MediaWiki Auth Script Per User Group

### PR DESCRIPTION
**Pull request by kingthorin**
Per
https://groups.google.com/forum/#!topic/zaproxy-users/ijDxvsZI6do
ensure Content-Length is set appropriately.

### CODE REVIEW

File: authentication/MediaWikiAuthentication.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ MWAuthenticator.prototype = {
 			requestInfo += ' with body: ' + requestBody;
 			msg.setRequestBody(requestBody);
 		}
+		msg.getRequestHeader().setContentLength(requestBody.length);
```
Comment (thc202) [inline authentication/MediaWikiAuthentication.js]: Better with `msg.getRequestBody().length()` (avoids byte vs char issues and good example for other scripts).

Comment (kingthorin) [reply authentication/MediaWikiAuthentication.js]: Ok I'll tweak that.

**Comment by thc202** [comment]
Do you plan to update the other scripts?

**Comment by kingthorin** [comment]
I was kind of after a quick hit. This was the one mentioned so I tackled it.

**Comment by thc202** [comment]
OK, we can tackle the others later.

**Pull request merged**